app/views/RegisterView.js

```diff
@@ -58,6 +60,10 @@ export default class RegisterView extends LoggedView {
 	}
 
 	shouldComponentUpdate(nextProps, nextState) {
+		const { validationError } = this.state;
```

Comment:
Instead of `validationError`, can we test a primitive?
Suggestion: add another state like `error: true/false` and add it to `shouldUpdateState` const